src/components/Swap.tsx

```diff
@@ -63,20 +63,19 @@ const Swap = () => {
   const anchorPetnames = [...instanceIds.keys()];
   const areAnchorsLoaded =
     anchorPetnames.length &&
-    anchorPetnames
-      .every(petname => {
-        const metrics = metricsIndex.get(petname);
-        const brand = metrics?.anchorPoolBalance.brand;
-        const brandInfo = brand && brandToInfo.get(brand);
-        const governedParams = governedParamsIndex.get(petname);
-        return metrics && brandInfo && governedParams;
-      })
+    anchorPetnames.every(petname => {
+      const metrics = metricsIndex.get(petname);
+      const brand = metrics?.anchorPoolBalance.brand;
+      const brandInfo = brand && brandToInfo.get(brand);
+      const governedParams = governedParamsIndex.get(petname);
+      return metrics && brandInfo && governedParams;
```

Comment:
minor, but this does all the gets even when we know one failed and the callback fails. it's a tradeoff with readability of course

Reply:
True, really the only extra `get` is governedParams but still